frontend/src/pages/item/_itemId/index.vue

```diff
@@ -288,6 +294,8 @@ onMounted(async () => {
     })
   ).data;
 
+  selectSources.value = getItemizedSelect(item.value.MediaSources ?? []);
```

Comment:
The computed should live below all the declarations so we avoid reference errors.

Reply:
I think I'll just re-make the onMounted part to avoid variables initialisation like this

Reply:
@ferferga fixed the order by removing the onMounted logic